src/main/java/gift/response/ResponseProductId.java

```diff
@@ -0,0 +1,15 @@
+package gift.response;
+
+public class ResponseProductId {
+    
+    private Long product_id;
```

Comment:
java에서의 변수명은 [카멜케이스](https://ko.wikipedia.org/wiki/%EC%B9%B4%EB%A9%9C_%ED%91%9C%EA%B8%B0%EB%B2%95)가 컨벤션입니다.   
```
Non-constant field names (static or otherwise) are written in [lowerCamelCase]
```
https://google.github.io/styleguide/javaguide.html#s5.2.5-non-constant-field-names